### ISSUE TITLE
LDAP lockout issue

### ISSUE THREAD

**Issue by frbapolkosnik** [crits/crits#445]
If you have LDAP auth set up and an user gets himself locked out on LDAP, he will also disable user's account in CRITs if he is trying to login while locked out. This might be quite a headache when you got a lot of active users.

**Comment by wxsBSD** [comment]
Does it make sense to disable lockout on crits accounts if using leap to auth, on the assumption that the leap server will lock the account?

**Comment by apolkosnik** [comment]
Once they get their LDAP unlocked, the CRITs admin needs to enable the
account(s)

**Comment by wxsBSD** [comment]
Right. And my question is if we should avoid that by not disabling an account if ldap auth is being used. It sounds like a good way to shoot ourselves in the foot though.

**Comment by apolkosnik** [comment]
Perhaps the best way would be appropriate error message during the logon
displayed to the user.

**Comment by wxsBSD** [comment]
As long as the message is the same for bad password and locked account, I'm fine with this. Attackers should not be able to distinguish between the two.
